kujira/chain.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "../../chain.schema.json",
+    "chain_name": "kujira",
+    "chain_id": "kaiyo-1",
+    "pretty_name": "Kujira",
+    "status": "live",
+    "network_type": "mainnet",
+    "bech32_prefix": "kujira",
```

Comment:
Can we please register this prefix to SLIP 173?
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done: https://github.com/satoshilabs/slips/pull/1356